client/src/main/java/io/pravega/client/connection/impl/ClientConnection.java

```diff
@@ -69,4 +70,9 @@ interface CompletedCallback {
         void complete(ConnectionFailedException e);
     }
 
+    /**
+     * Get the endpoint details of a segment.
```

Comment:
description can be changed to "Get the location details of a segment" to be in consistent with other terms in the file.

Reply:
Incorporated the review comment.